ui/Screen/Org/Projects.svelte

```diff
@@ -22,7 +22,12 @@
   const select = event => {
     const orgProject = event.detail;
     if (orgProject.maybeProject) {
-      push(path.projectSource(orgProject.maybeProject.id));
+      push(
+        path.projectSource(
+          orgProject.maybeProject.id,
+          orgProject.maybeProject.default_user.id
```

Comment:
So here it looks like the project source indeed depends on a user. But I was under the impression that source code is tied to the device/peer, not the user. How do we go from `user -> device -> repo`?

Reply:
Same concern as in https://github.com/radicle-dev/radicle-upstream/pull/620#pullrequestreview-440583382

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/d5b10c171dddbf5b93c4a3b1493d25c1689799f0